client/src/components/CommentBody/CommentBody.css

```diff
@@ -1,19 +1,66 @@
+body {
+  text-align: center;
+}
+
 .CommentBody {
   display: flex;
   flex-direction: row;
   justify-content: space-between;
   align-items: flex-start;
-  width: 60%;
+  width: 50rem;
   height: auto;
   background-color: var(--gray-dark);
   font-size: 14px;
   font-family: var(--paragraph-font);
+  padding: 1.2rem;
+}
+
+.CommentBody p {
+  text-align: left;
+}
+
+.CommentBody_item {
+  margin: 0.7rem;
+  text-align: left;
+  height: 90%;
+  margin-bottom: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  height: 9rem;
+}
+
+.CommentBody_center_item {
+}
+
+.CommentBody_name {
+  text-align: center !important;
+  margin-bottom: 0;
+  font-size: 18px;
+}
+.CommentBody_activity {
+  text-align: left;
+  margin-right: 0.8rem;
+  margin-top: 0.5rem;
+  display: flex;
 }
-.CommentBody__date {
-  margin-left: 10px;
+
+.Date {
```

Comment:
`.Date` is a pretty greedy selector (meaning it could end up applying to a lot of places. Can you do something more inline with the naming convention? `.CommentBody__date` is good (the one deleted)